### ISSUE TITLE
missing includes for PIE517 support with base 7 and synApps 5.8

### ISSUE THREAD

**Issue by prjemian** [epics-modules/motor#89]
* arch: linux-x86_64
* base: 7.0.1.1
* synApps: 5.8
* motor 6.10

compiler presented this error:

```
username@host .../motorApp/PiSrc $ make
make -C O.linux-x86_64 -f ../Makefile TOP=../../.. \
    T_A=linux-x86_64 install
make[1]: Entering directory '/usr/local/epics/synApps/support/motor-R6-10/motorApp/PiSrc/O.linux-x86_64'
/usr/bin/g++  -D_GNU_SOURCE -D_DEFAULT_SOURCE           -D_X86_64_  -DUNIX  -Dlinux     -O3 -g   -Wall      -mtune=generic     -m64 -fPIC -I. -I../O.Common -I. -I. -I.. -I../../../include/compiler/gcc -I../../../include/os/Linux -I../../../include      -I/home/username/Apps/epics/synApps/support/asyn-R4-32/include   -I/home/username/Apps/epics/synApps/support/seq-2-2-5/include   -I/home/username/Apps/epics/synApps/support/busy-R1-7/include -I/usr/local/epics/base-7.0.1.1/include/compiler/gcc -I/usr/local/epics/base-7.0.1.1/include/os/Linux -I/usr/local/epics/base-7.0.1.1/include   -I/home/username/Apps/epics/synApps/support/ipac-2-14/include        -c ../devPIE517.cc
../devPIE517.cc: In function ‘RTN_STATUS PIE517_build_trans(motor_cmnd, double*, motorRecord*)’:
../devPIE517.cc:309:58: error: ‘errlogMessage’ was not declared in this scope
  errlogMessage("PIE517_build_trans(): buffer overflow.\n");
                                                          ^
/usr/local/epics/base-7.0.1.1/configure/RULES_BUILD:233: recipe for target 'devPIE517.o' failed
make[1]: *** [devPIE517.o] Error 1
make[1]: Leaving directory '/usr/local/epics/synApps/support/motor-R6-10/motorApp/PiSrc/O.linux-x86_64'
/usr/local/epics/base-7.0.1.1/configure/RULES_ARCHS:58: recipe for target 'install.linux-x86_64' failed
make: *** [install.linux-x86_64] Error 2
```

**Comment by prjemian** [comment]
Some includes are missing for `errlogMessage`, `malloc`, and `atof`

These additions will allow the build to complete.

```
devPIE517.cc:#include <errlog.h> /* fix */
drvPIE517.cc:#include <stdlib.h> /* fix */
drvPIE517.cc:#include <errlog.h> /* fix */
```

**Comment by prjemian** [comment]
This was fixed previously in  8409249

**Comment by prjemian** [comment]
But that was for the Delta Tau support.
